src/modules/infobox/InfoboxModule.vue

```diff
@@ -58,14 +65,52 @@ function onPrint() {
     promptUserToPrintHtmlContent(content.value)
 }
 function onClose() {
-    store.dispatch('clearAllSelectedFeatures', dispatcher)
-    store.dispatch('clearClick', dispatcher)
+    if (showFeatureInfoInBottomPanel.value) {
+        store.dispatch('clearAllSelectedFeatures', dispatcher)
+        store.dispatch('clearClick', dispatcher)
+    } else if (showElevationProfile.value) {
+        // if feature details are shown in the floating tooltip we don't want to close the detail
+        // when clicking on the X button, but only close the profile
+        onHideProfile()
+    }
+}
+function onHideProfile() {
+    store.dispatch('setProfileFeature', { feature: null, ...dispatcher })
 }
 </script>
 
 <template>
-    <div v-if="showContainer" class="infobox card rounded-0" data-cy="infobox" @contextmenu.stop>
-        <div class="infobox-header card-header d-flex justify-content-end" data-cy="infobox-header">
+    <div
+        v-if="showContainer"
+        class="infobox card rounded-0 clear-no-ios-long-press"
+        data-cy="infobox"
+        @contextmenu.stop
```

Comment:
Is this needed? It's just a div, where does the event come from?

Reply:
that was already there, my linter just made one line for each entry.

I think it's there to not have the context menu when right clicking on the whole Infobox, don't know if it is really useful :face_with_diagonal_mouth: